relay-server/src/services/project/metrics/post.rs

```diff
@@ -0,0 +1,283 @@
+use relay_dynamic_config::{ErrorBoundary, Feature, Metrics};
+use relay_filter::FilterStatKey;
+use relay_metrics::{Bucket, MetricNamespace};
+use relay_quotas::Scoping;
+use relay_system::Addr;
+
+use crate::metric_stats::MetricStats;
+use crate::services::outcome::{Outcome, TrackOutcome};
+use crate::services::project::ProjectState;
+use crate::utils;
+
+use super::{state, Transformer};
+
+/// Applies an initial filter to incoming metrics using the project state.
+pub struct PostProjectTransform<'a> {
```

Comment:
Would be nice to give this a descriptive name (i.e. what it does, not when it's called).